src/Microsoft.Health.Fhir.Core/Messages/Export/CreateExportRequest.cs

```diff
@@ -18,6 +19,18 @@ public CreateExportRequest(Uri requestUri)
             RequestUri = requestUri;
         }
 
+        public CreateExportRequest(Uri requestUri, string destinationType, string destinationConnectionString)
+        {
+            EnsureArg.IsNotNull(requestUri, nameof(requestUri));
+            EnsureArg.IsNotNullOrWhiteSpace(destinationType, nameof(destinationType));
+            EnsureArg.IsNotNullOrWhiteSpace(destinationConnectionString, nameof(destinationConnectionString));
+
+            RequestUri = requestUri;
+            DestinationInformation = new DestinationInfo(destinationType, destinationConnectionString);
+        }
+
         public Uri RequestUri { get; }
+
+        public DestinationInfo DestinationInformation { get; }
```

Comment:
nit: DestinationInfo? .NET library seems to use Info instead of Information (e.g., MethodInfo, PropertyInfo).

Reply:
Wanted to avoid having `DestinationInfo DestinationInfo` in the code since it might be a bit confusing.

Reply:
Read up on the guidelines and updated this to `DestinationInfo DestinationInfo`